app/src/main/java/com/peacecorps/pcsa/circle_of_trust/MessageAdapter.java

```diff
@@ -0,0 +1,56 @@
+package com.peacecorps.pcsa.circle_of_trust;
+
+import android.app.Dialog;
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.peacecorps.pcsa.R;
+
+/**
+ * This adapter is to initialise views of the customised dialog box which gets invoked on pressing Help me button.
+ *
+ * @author Rohan
+ * @since 25-02-2016
```

Comment:
Oh yes, it is. I shall make the necessary changes after your complete review so that I can commit once